targets/targets.json

```diff
@@ -12762,6 +12762,38 @@
             "IAR"
         ],
         "config": {
+            "spim-ccm-enable": {
+                "help": "Enable SPIM CCM mode to spare 32KiB SRAM for normal use",
+                "value": 0
+            },
+            "usb-uart": {
+                "help": "Configure USB_UART. USB_UART and USB_UART_TX/USB_UART_RX must be consistent.",
+                "value": null
+            },
+            "usb-uart-tx": {
+                "help": "Configure USBTX. USB_UART and USBTX/USBRX must be consistent.",
+                "value": null
+            },
+            "usb-uart-rx": {
+                "help": "Configure USBRX. USB_UART and USBTX/USBRX must be consistent.",
+                "value": null
+            },
+            "stdio-uart": {
+                "help": "Configure STDIO_UART. STDIO_UART and STDIO_UART_TX/STDIO_UART_RX must be consistent. STDIO_UART defaults to USB_UART.",
+                "value": null
+            },
+            "stdio-uart-tx": {
+                "help": "Configure STDIO_UART_TX. STDIO_UART and STDIO_UART_TX/STDIO_UART_RX must be consistent. STDIO_UART_TX defaults to USBTX.",
+                "value": null
+            },
+            "stdio-uart-rx": {
+                "help": "Configure STDIO_UART_RX. STDIO_UART and STDIO_UART_TX/STDIO_UART_RX must be consistent. STDIO_UART_RX defaults to USBRX.",
+                "value": null
+            },
+            "custom-pinname-file": {
```

Comment:
what is the use case for this, why do we need this "header file name" here? I dont recall seeing this before in other targets, the overwrite is done differently?

Reply:
Removed `custom-pinname-file`. Re-organize `PinNames.h` so that custom `PinNames.h` can place in custom target (`TARGET_`) folder.